packages/core/src/initialization/anchoring.ts

```diff
@@ -40,22 +40,17 @@ export class UnusableAnchorChainsError extends Error {
   }
 }
 
-export class CustomMainnetCasError extends Error {
-  constructor() {
-    super('Cannot use custom anchor service on Ceramic mainnet')
-  }
-}
-
 const TRAILING_SLASH = new RegExp(/\/+$/g) // slash at the end of the string
 const MAINNET_CAS_URLS = [
```

Comment:
can we add the new URL for gitcoin in here while we're at it so they won't get the warning?